packages/default/scss/pager/_layout.scss

```diff
@@ -199,9 +201,6 @@
         }
     }
 
-
-
-
     // Responsive
     .k-pager-lg,
```

Comment:
There are still styles for k-pager-lg, based on the design and our conversations, we no longer need these styles, please remove them.